client/my-sites/email/titan-add-mailboxes/titan-mailbox-pricing-notice.tsx

```diff
@@ -43,25 +47,32 @@ function getPriceMessage( props ) {
 		  );
 }
 
-function getPriceMessageExplanation( props ) {
-	const { purchaseCost, renewalCost, translate } = props;
-
+function getPriceMessageExplanation( { purchaseCost, renewalCost, titanMailProduct, translate }: { purchaseCost: EmailCost, renewalCost: EmailCost, titanMailProduct: ProductListItem, translate: typeof originalTranslate } ): TranslateResult {
 	// We don't need any explanation of the price at this point, because we have already handled it previously.
 	if ( purchaseCost.amount === 0 ) {
 		return '';
 	}
-	return purchaseCost.amount < renewalCost.amount
-		? translate(
+
+	if ( purchaseCost.amount < renewalCost.amount ) {
+		return isTitanMonthlyProduct( titanMailProduct )
+			? translate(
 				'This is less than the regular price because you are only charged for the remainder of the current month.'
-		  )
+			)
+			: translate(
+				'This is less than the regular price because you are only charged for the remainder of the current year.'
+			);
+	}
+
+	return isTitanMonthlyProduct( titanMailProduct )
+		? translate(
+			'This is more than the regular price because you are charged for the remainder of the current month plus any additional month until renewal.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'This is more than the regular price because you are charged for the remainder of the current month plus any additional month until renewal.' )` *ES Score: 8*